docs/userguide/dockervolumes.md

```diff
@@ -216,7 +216,8 @@ then mounting the volumes from the `dbstore` container hides the
 from the `dbstore` container are visible.
 
 You can use multiple `--volumes-from` parameters to bring together multiple data
```

Comment:
@moxiegirl  you asked:
Can't the destination be a dir/file situation too?


From what I tested it only allow a absolute path, Not quite know you mean file here. Could you double check ?